### PR TITLE
Firespread fix

### LINKED ISSUE #194: Fire not spreading
**Describe the bug**
Fire does not spread once moved.

### PR DESCRIPTION
**Pull request by eirikh1996**
### Describe in detail what your pull request acomplishes
In this pull request, I fix a problem with fire not spreading on moving crafts

### Related issues:
- Fixes #194 

### Checklist
- [x] Unit tests 
- [x] Proper internationalization
- [x] Compiled/tested

### CODE REVIEW

File: modules/Movecraft/src/main/java/net/countercraft/movecraft/listener/BlockListener.java [hunk 1]
```diff
@@ -405,4 +410,14 @@ public void run() {
         }
 
     }
+
+    private Craft adjacentCraft(Location location) {
```
Comment (cccm5) [inline modules/Movecraft/src/main/java/net/countercraft/movecraft/listener/BlockListener.java]: Use nullability annotations on this (return type and parameters)

File: modules/v1_10_R1/src/main/java/net/countercraft/movecraft/compat/v1_10_R1/IWorldHandler.java [hunk 2]
```diff
@@ -131,7 +130,18 @@ public void rotateCraft(@NotNull Craft craft, @NotNull MovecraftLocation originP
...
-        //*       Step six: Send to players       *
+        //*   Step six: Process fire spread         *
+        //*******************************************
+        for (BlockPosition position : rotatedPositions.values()) {
+            IBlockData type = nativeWorld.getType(position);
+            if (!(type.getBlock() instanceof BlockFire)) {
+                continue;
+            }
+            BlockFire fire = (BlockFire) type.getBlock();
+            fire.b(nativeWorld, position, type, new Random());
```
Comment (cccm5) [inline modules/v1_10_R1/src/main/java/net/countercraft/movecraft/compat/v1_10_R1/IWorldHandler.java]: Should this be a new Random instance or the random instance from the world? (for all of the occurrences)

**Comment by eirikh1996** [comment]
Requested changes made @cccm5

File: modules/Movecraft/src/main/java/net/countercraft/movecraft/listener/BlockListener.java [hunk 3]
```diff
@@ -57,7 +57,9 @@
 import org.bukkit.material.Attachable;
 import org.bukkit.material.MaterialData;
 import org.bukkit.scheduler.BukkitRunnable;
+import org.jetbrains.annotations.NotNull;
 
+import javax.annotation.Nullable;
```
Comment (cccm5) [inline modules/Movecraft/src/main/java/net/countercraft/movecraft/listener/BlockListener.java]: this import is dangerous

**Comment by eirikh1996** [comment]
Requested change made @cccm5

**Pull request merged**